federation/cmd/federation-controller-manager/app/controllermanager.go

```diff
@@ -54,6 +58,13 @@ import (
 	"k8s.io/client-go/discovery"
 )
 
+const (
+	apiserverWaitTimeout   = 30 * 2 * time.Second
+	apiserverRetryInterval = 1 * time.Second
+
+	federationOnlyNamespace = "federation"
```

Comment:
Agree with your comment. I would make it configurable. Also what could be the name for  such a namespace in case of default. is `federation` name fine? or should i changed it to something like `federation-only`